lib/index.js

```diff
@@ -10,6 +10,7 @@ function RpcClient(opts) {
   this.port = opts.port || 9998;
   this.user = opts.user || 'user';
   this.pass = opts.pass || 'pass';
+  this.timeout = opts.timeout || null;
```

Comment:
Should this default to 0 (disabled) or some other number instead? I'm just wondering what happens below if this is set to `null`, would it trigger some exception?

Reply:
Not sure, that `0` or `null` is the default for https://nodejs.org/api/https.html#httpsrequestoptions-callback. I would probably just keep `undefined`.